src/emoji/emojiSelectors.js

```diff
@@ -27,3 +27,11 @@ export const getActiveRealmEmojiTransformed = createSelector(getActiveRealmEmoji
   });
   return transformedEmojis;
 });
+
+export const getAllRealmEmojiTransformed = createSelector(getAllRealmEmoji, emojis => {
+  const transformedEmojis = {};
```

Comment:
Use `reduce`. This will combine all three operators in a single, more atomic operation.
The code will be almost the same. Have a look at many other selectors that use it, if you are unsure how to implement.